roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -13,6 +13,11 @@ kubeletArguments: {{ openshift.node.kubelet_args | to_json }}
 {% endif %}
 masterKubeConfig: system:node:{{ openshift.common.hostname }}.kubeconfig
 networkPluginName: {{ openshift.common.sdn_network_plugin_name }}
+# networkConfig struct introduced in origin 1.0.6 and OSE 3.0.2 which
```

Comment:
I'll verify real quick, my experience has been that invalid keys are ignored. Invalid values for known keys are where things blow up.